src/utils/cli.ts

```diff
@@ -0,0 +1,51 @@
+import type { Command } from "@gemini-testing/commander";
+import logger from "./logger";
+import { requireModule } from "./module";
+
+// used from https://github.com/sindresorhus/escape-string-regexp/blob/main/index.js
+const escapeRe = (str: string): string => {
```

Comment:
Function which is used from escape-string-regexp. Because there are no typings for it (for version with support commonjs).

Reply:
Hmm is there a reason we can't use `_.escapeRegExp` ?